Scripts/BuildPhases/GenerateCredentials.sh

```diff
@@ -65,28 +61,14 @@ SECRETS_DESTINATION_FILE="${BUILD_DIR}/Secrets/Secrets.swift"
 mkdir -p $(dirname "$SECRETS_DESTINATION_FILE")
 
 # If the WordPress Production Secrets are available for WordPress, use them
-if [ -f "$PRODUCTION_SECRETS_FILE" ] && [ "$BUILD_SCHEME" == "WordPress" ]; then
+if [ -f "$WORDPRESS_SECRETS_FILE" ] && [ "$WP_BUILD_SCHEME" == "WordPress" ]; then
```

Comment:
Based on how this variable is used, would the built-in `TARGET_NAME` be more appropriate? It seems a bit strange to hardcode a scheme variable `WP_BUILD_SCHEME` in the build configuration files, considering the relationship between the Xcode scheme and the build configuration is not one-to-one.

Reply:
Great idea. I didn't think of that. Yeah, there is no need for `WP_BUILD_SCHEME` now. Updating it.

Reply:
It turned out not to be as simple.

For WordPress, the "GenerateCredentials" target is used to generate secrets. For this target, `TARGET_NAME` is `GenerateSecrets`.

<img width="243" alt="Screenshot 2025-03-25 at 8 27 10 AM" src="https://github.com/user-attachments/assets/84d8e2fb-b1a0-45f6-aff6-48d1e11a9286" />

The same target is **not** used for Jetpack that instead runs the `GenerateCredentials.sh` script directly:

<img width="870" alt="Screenshot 2025-03-25 at 8 27 22 AM" src="https://github.com/user-attachments/assets/6349299f-e01b-4b09-ac64-151cff273110" />

Cool.

Reply:
I made the following changes:

- Remove the `GenerateCredentails` scheme; use a simple script for both apps.
- Remove `WP_BUILD_SCHEME` and use the existing `TARGET_NAME` env variable instead.